src/Core/Framework/DependencyInjection/app.xml

```diff
@@ -674,5 +675,12 @@
             <argument type="service" id="Doctrine\DBAL\Connection"/>
             <argument type="service" id="Shopware\Core\Framework\App\Privileges\Privileges"/>
         </service>
+
+        <service id="Shopware\Core\Framework\App\Subscriber\SystemLanguageChangedSubscriber">
```

Comment:
can we not put the subscriber into Administration bundle? It would fix also the on-invalid hack here

Reply:
you're totally right 👍